src/Raven.Server/Dashboard/DatabasesInfoRetriever.cs

```diff
@@ -241,27 +296,128 @@ public static IEnumerable<AbstractDashboardNotification> FetchDatabasesInfo(Serv
             }
 
             yield return databasesInfo;
+            yield return databasesOngoingTasksInfo;
             yield return indexingSpeed;
             yield return trafficWatch;
             yield return drivesUsage;
         }
 
-        private static long GetOngoingTasksCount(DocumentDatabase database)
+
+        private static long GetExtRepCountOnNode(DocumentDatabase database, DatabaseRecord dbRecord, ServerStore serverStore) 
```

Comment:
Instead of repeating all those methods you can use a single method like here:

```
private static long GetTaskCountOnNode<T>(DocumentDatabase database, DatabaseRecord dbRecord, ServerStore serverStore, IEnumerable<IDatabaseTask> tasks, Func<T, IDatabaseTaskStatus> getTaskStatus, Func<T, bool> filter = null)
        where T: IDatabaseTask
        {
            long taskCountOnNode = 0;
            foreach (var task in tasks)
            {
                if (filter != null && filter((T)task) == false)
                    continue;

                var state = getTaskStatus((T)task);
                var taskTag = database.WhoseTaskIsIt(dbRecord.Topology, task, state);
                if (serverStore.NodeTag == taskTag)
                {
                    taskCountOnNode++;
                }

            }

            return taskCountOnNode;
        }
```

and then call it like:

```
GetTaskCountOnNode<RavenEtlConfiguration>(database, dbRecord, serverStore, database.EtlLoader.RavenDestinations,
                (task) => EtlLoader.GetProcessState(task.Transforms, database, task.Name));


            GetTaskCountOnNode<PeriodicBackupConfiguration>(database, dbRecord, serverStore, database.PeriodicBackupRunner.PeriodicBackups.Select(x => x.Configuration),
                (task) => database.PeriodicBackupRunner.GetBackupStatus(task.TaskId), task => task.Name.StartsWith("Server Wide") == false);
```

Reply:
done (except for subscriptions)